mhubert-pmn/Front-Url-Shortener
# [Update] Align front and update components

## Task
Align front and update components

## Type of change

## Description
- Align indentation
- Update Button, Layout, Form and Header components
- Fix style

## Testing scenario
In the terminal:
- Launch and build: `docker-compose up --build
- Go to `http://localhost:3000/inscription` and check if the header, the layout and the form appear correctly
- Change the size of your screen to see the responsive style

## Trello card
[https://trello.com/c/THKQDkJh](https://trello.com/c/THKQDkJh)
